docs/product-getting-started-guide/src/main/asciidoc/decision-table-create-proc.adoc

```diff
@@ -0,0 +1,107 @@
+[id='dec-table-create-proc']
+= Creating a speed violation guided decision table
+
+The following chapter instructs you how to create a speed violation decision table in {PRODUCT} using the Guided Decision Table wizard.
+
+[float]
+== Prerequisite
+
+Created both the Violation and Driver data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Guided Decision Table*, then enter:
+
+* *Guided Decision Table*: `SpeedViolationRule`
+* *Package*: `org.jboss.example.traffic_violations`
++
+
+. Select *Unique Hit* from the *Hit Policy* pull-down menu.
+. Select *Extended entry, values defined in table body* in the *Table format* section.
+. Click *Ok* to open the *Guided Decision Tables* designer.
++
+
+.Guided Decision Tables designer
+image::dec-table.png[]
+
+== Inserting Violation columns
+
+. Click *Columns* -> *Insert Column* and select *Include advanced options*.
+. Select *Add an Attribute column* and click *Next*.
++
+
+.Add a new column window
+image::new-column.png[]
+. Select *Ruleflow-group* and click *Finish*.
+. Expand *Attribute columns* and enter `trafficViolation` in the *Default value* field.
++
+
+.Attribute columns window
+image::attrib-column.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* -> *+Create a new Fact Pattern*.
+. Select *Violation* from the *Fact type* pull-down menu, enter `v` in the *Binding* field, and click *+OK*.
++
+
+.Create a new fact pattern window
+image::fact-pattern.png[]
+. Select *Calculation type* -> *Literal value*.
+. Select *Field* and then *type* from the *Field* pull-down menu.
+. Select *Operator* and then *equal to* from the *Operator* pull-down menu.
+. Select *Value options* and enter `Driving while intoxicated,DWI=Driving while under the influence of drugs` in the *Value list (optional)* field.
+. Select *Additional info*, enter `Violation Type` in the *Header (description)* field, and click *Finish*.
++
+
+.Violation Type header
+image::violation-columns.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
```

Comment:
"... -> **Pattern**, and select .."  Comma (serial), for consistency. Yep, I flagged a comma :)

Reply:
You are now referred to as eagle eye - done :)